editor/src/clj/editor/search_results_view.clj

```diff
@@ -307,3 +328,267 @@
                           (update-search-results! search-results-view results-tab-tree-view progress-indicator)
                           (show-search-results-tab-fn))]
     (start-search-in-files! project prefs results-tab-tree-view progress-indicator open-fn show-matches-fn)))
+
+(defn- resource-cell [{:keys [resource qualifier]}]
+  {:graphic {:fx/type fx.h-box/lifecycle
+             :spacing 6
+             :children (cond-> [{:fx/type fxui/label
+                                 :text (resource/resource->proj-path resource)}]
+                               qualifier
+                               (conj {:fx/type fxui/label
+                                      :style {:-fx-text-fill :-df-text-darker}
+                                      :text qualifier}))}})
+
+(defn- property->edit-type-dispatch-value [property]
+  (let [edit-type (-> property :edit-type :type (or (:type property)))]
+    (or (g/value-type-dispatch-value edit-type)
+        edit-type)))
+
+(defmulti property-column-suffix property->edit-type-dispatch-value)
+(defmethod property-column-suffix :default [_] "")
+(defmethod property-column-suffix types/Vec2 [property]
+  (str ": " (string/join " " (:labels property ["X" "Y"]))))
+(defmethod property-column-suffix types/Vec3 [property]
+  (str ": " (string/join " " (:labels property ["X" "Y" "Z"]))))
+(defmethod property-column-suffix types/Vec4 [property]
+  (str ": " (string/join " " (:labels property ["X" "Y" "Z" "W"]))))
+
+(defmulti override-value-cell-view property->edit-type-dispatch-value)
+
+(defn- overridden-style-classes [property]
+  (if (contains? property :original-value)
+    ["overridden"]
+    []))
```

Comment:
Would it work to return `nil` instead of an empty list here? Nil is also an empty collection, right?